### ISSUE TITLE
Drop `Muster List` from Vault Edit

### ISSUE THREAD

**Issue by IanMayo** [DeepBlueCLtd/RCO#241]
We create must forms from the `VaultList` component.

We can delete it from the `VaultEdit` component

![image](https://user-images.githubusercontent.com/1108513/236462963-6988649c-7905-461a-a68b-113a73e6808f.png)

**Comment by TahaKhanAbdalli** [comment]
Should I also delete the related components that are no longer needed?

**Comment by IanMayo** [comment]
> Should I also delete the related components that are no longer needed?

Yes please - if you can see they are not referred to from anywhere else.
